docs/localdatetime-draft.md

```diff
@@ -0,0 +1,95 @@
+
+**API**: The API looks something like this:
+
+```typescript
+class Temporal.LocalDateTime {
+  // creation
+  constructor(absolute: Temporal.Absolute, timeZone: Temporal.TimeZone, calendar?: Temporal.Calendar);
+  static from(item: string | object, options?: object) : Temporal.LocalDateTime;
+
+  // immutable 'mutation'
+  with(localDateTimeLike: object, options?: object) : Temporal.LocalDateTime;
+  withCalendar(calendar: Temporal.Calendar) : Temporal.LocalDateTime;
+
+  // getters / field access
+  get era(): string | undefined;
+  get year(): number;
+  get month(): number;
+  get day(): number;
+  get hour(): number;
+  get minute(): number;
+  get second(): number;
+  get millisecond(): number;
+  get microsecond(): number;
+  get nanosecond(): number;
+  get timeZone(): Temporal.TimeZone;
+  get timeZoneOffsetNanoseconds(): number;
+  get timeZoneOffsetString(): string;
+  get calendar(): Temporal.Calendar;
+  get dayOfWeek(): number;
+  get dayOfYear(): number;
+  get weekOfYear(): number;
+  get hoursInDay(): number;
+  get daysInMonth(): number;
+  get daysInYear(): number;
+  get isLeapYear(): boolean;
+  getFields(): object;
+  getISOCalendarFields(): object;
+
+  // type conversion
+  toAbsolute(): Temporal.Absolute;
+  toDateTime(): Temporal.DateTime;
+  toDate(): Temporal.Date;
+  toYearMonth(): Temporal.YearMonth;
+  toMonthDay(): Temporal.MonthDay;
+  toTime(): Temporal.Time;
+
+  // comparison
+  static compare(one: Temporal.LocalDateTime, two: Temporal.LocalDateTime): number;
+  equals(other: Temporal.LocalDateTime): boolean;
+
+  // arithmetic
+  plus(duration: Temporal.Duration | object, options?: object) : Temporal.LocalDateTime;
+  minus(duration: Temporal.Duration | object, options?: object) : Temporal.LocalDateTime;
+  difference(other: Temporal.LocalDateTime, options?: object) : Temporal.Duration;
+
+  // serialization / presentation
+  toString() : string;
+  toJSON() : string;
+  toLocaleString(locales?: string | Array<string>, options?: object): string;
+}
+```
+
+(This is not a fully functional description, it's made concise for readability.
+For example, the details of the options bags are omitted.)
+
+It basically has all the API of DateTime, but with a time zone as well.
+
+To create one, you use `Temporal.LocalDateTime.from`.
+- From a string: `from(string)`
+- From an Absolute: `from({ absolute, timeZone })`
+- From a DateTime, which potentially needs disambiguation: `from({ dateTime, timeZone }, { disambiguation })` or possibly `from({ ...dateTime.getFields(), timeZone }, { disambiguation })`
+- From raw DateTime fields: `from({ year, month, day, etc., timeZone })`
+
+As in the other Temporal types, the constructor is more low-level and takes an Absolute, a TimeZone, and an optional Calendar.
```

Comment:
I strongly disagree with any notion of the time zone being required but the calendar being optional.  They should either both have defaults (UTC/ISO) or neither have defaults.

Reply:
Because the Absolute+TimeZone tuple is an implementation detail, and the calendar is really only present as a part of the DateTime I really disagree with this statement prima facie.

Reply:
I'm skeptical that insisting on this will change anyone's mind about the default calendar. Time zones and calendars are somewhat analogous, but not 100%. But I'm fine to write this very high-level document such that it doesn't indicate one way or the other :smile: